modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -8,7 +8,9 @@ import java.util.ArrayList;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import javax.inject.Inject;
-
+{{#isResponseFile}}
+import java.io.FileInputStream;
```

Comment:
this will not work because you need to be inside an operation `{{#operation}}` to be able to access this information, it's not a "class" information. To prevent FileInputStream to end up there, I will need to make this dynamic in the JavaPlayFramework codegen and it will not be handled in the mustache file.